docs/docs/authenticate/generic-oidc.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Generic OIDC Provider
+sidebar_position: 10
+---
+
+The example below details the config settings required by Firezone to enable SSO
+through a generic OIDC provider. The configuration file can be found at
+`/etc/firezone/firezone.rb`. To pick up changes, run `firezone-ctl reconfigure`
+and `firezone-ctl restart` to update the application.
+
+```ruby
+# This is an example using Google and Okta as an SSO identity provider.
```

Comment:
Thanks for adding this explanation. Probably good to update the config file template at `omnibus/cookbooks/firezone/attributes/default.rb` with this info. More users probably reading through that.